code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -684,7 +684,7 @@
 			gain = 100
 		if(mind.assigned_role == JOB_NAME_CLOWN)
 			gain = rand(-1000, 1000)
-	investigate_log("([key_name(src)]) has been consumed by the singularity.", INVESTIGATE_ENGINES) //Oh that's where the clown ended up!
+	usr.investigate_log("has been consumed by the singularity.", INVESTIGATE_ENGINES) //Oh that's where the clown ended up!
```

Comment:
this was inadvertently changed from src

Reply:
Good catch.

Fixed in 4c1d911

Reply:
Actually, why is this even here? Its just duplicate code of /mob/living/singularity_act()

Reply:
Ive retooled it in d6052e8